src/graphql/admin/mongoose_graphql_server_admin_query.erl

```diff
@@ -0,0 +1,20 @@
+-module(mongoose_graphql_server_admin_query).
+-behaviour(mongoose_graphql).
+
+-export([execute/4]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+
+execute(_Ctx, server, <<"status">>, _) ->
+    case mongoose_server_api:status() of
+        {ok, String} ->
+            {ok, #{<<"statusCode">> => <<"RUNNING">>, <<"message">> => String}};
+        {_, String} ->
+            {ok, #{<<"statusCode">> => <<"NOT_RUNNING">>, <<"message">> => String}}
```

Comment:
Maybe this could be checked somehow?

Reply:
I tried to check it, but when mongooseim is not running something like this is returned: Failed RPC connection to the node mongooseim3@localhost: nodedown\n\nCommands to start a MongooseIM node:\n  start           Start a MongooseIM node as daemon (detached from terminal)\n  debug           Attach an interactive Erlang shell to a running MongooseIM node\n  live            Start MongooseIM node in live (interactive) mode\n  foreground      Start MongooseIM node in foreground (non-interactive) mode\nMongooseIM cluster management commands:\n  join_cluster other_node_name                Add current node to cluster\n  leave_cluster                               Make the current node leave the cluster\n  remove_from_cluster other_node_name         Remove dead node from the cluster\nExtra Commands:\n  bootstrap           Executes MongooseIM init scripts (used for initial configuration)\n  print_install_dir   Prints path to MongooseIM release directory\n  escript             Runs escript command using embedded Erlang Runtime System\n\n"

Reply:
I tried to cover it in a "stop_node_test" after the node was being stopped.